core/exception/backtrace_spec.rb

```diff
@@ -47,14 +47,14 @@
     end
   end
 
-  it "produces a backtrace for an exception captured using $!" do
-    exception = begin
+  it "captures the backtrace for an exception into $@" do
+    backtrace = begin
       raise
     rescue RuntimeError
-      $!
+      $@
     end
 
-    exception.backtrace.first.should =~ /backtrace_spec/
+    backtrace.first.should =~ /backtrace_spec/
```

Comment:
Corrects to test $@.

Reply:
This would be better as a separate example, it's still useful to test with `$!`

Reply:
Done.